mmdet/core/hook/set_epoch_hook.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.parallel import is_module_wrapper
+from mmcv.runner import HOOKS, Hook
+
+
+@HOOKS.register_module()
+class SetEpochHook(Hook):
```

Comment:
1. SetEpochInfoHook
2. Add a short description/summary in docstring though it is simple